src/janitor.c

```diff
@@ -160,7 +160,10 @@ static void launch_recheck(PgPool *pool)
 		/* send test query, wait for result */
 		slog_debug(server, "P: checking: %s", q);
 		change_server_state(server, SV_TESTED);
-		SEND_generic(res, server, PqMsg_Query, "s", q);
+		if (strcmpeq(q, "<empty>"))
```

Comment:
Instead of doing this check over and over in the janitor, I'd rather change the config parsing to handle this as a special case and store it as an the empty string.